style.css

```diff
@@ -386,13 +408,12 @@ footer {
 }
 
 .features {
-  padding-top: 5%;
+  padding-top: 1%;
   display: flex;
   justify-content: center;
   align-items: center;

```

Comment:
- [ ] Use `d-flex justify-content-center align-items` Bootstrap classes instead of these.